tutorials/tour/abstract-types.md

```diff
@@ -5,11 +5,10 @@ title: Abstract Types
 disqus: true
 
 tutorial: scala-tour
-num: 2
-outof: 35
```

Comment:
Why'd you remove the `outof` here? I see there's a second one (below) which doesn't agree with this one.